modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/contract/JobsSearchSettings.java

```diff
@@ -18,6 +20,13 @@
     private String timeFrom;
     private String timeTo;
 
+    public boolean areFiltersEmpty() {
+        if (isBlank(defaultString(name) + defaultString(activity) + defaultString(status) + defaultString(timeFrom) + defaultString(timeTo))) {
```

Comment:
first of all this condition can be written in 1 line, secondly check if this strings are blank separately, do not use concatenation